pkg/cmd/grafana-server/server.go

```diff
@@ -10,38 +10,36 @@ import (
 	"path/filepath"
 	"strconv"
 	"time"
+	// self registering services
```

Comment:
This comment can be removed

Reply:
Oh yeah, I use a plugin for vscode now that changes sorting of imports

Reply:
Done!